medic/medic-appium.js

```diff
@@ -222,6 +223,37 @@ function getLocalCLI(appPath) {
     return "./cordova";
 }
 
+function getConfigPath(appPath) {
+    return path.join(appPath, "config.xml");
+}
+
+function parseElementtreeSync(filename) {
+    var contents = fs.readFileSync(filename, util.DEFAULT_ENCODING);
+    if(contents) {
```

Comment:
If the file is empty the rest of the code is not valid anymore. Maybe throw an exception here if there is `!contents`?